manifest.json

```diff
@@ -0,0 +1,23 @@
+{
+  "browser_action": {
```

Comment:
add a `default_title` inside `browser_action` It will help to user to know about the extension while hovering above the extension icon in the browser tool bar